src/jacobi.h

```diff
@@ -0,0 +1,32 @@
+#ifndef _JACOBI_LIB_
+#define _JACOBI_LIB_
+
+#include <math.h>
+#include <stdlib.h>
+#include <string.h>
+
+#ifndef DIGITS_PRECISION
+    #define DIGITS_PRECISION 15
+#endif
+
+typedef double real_number;
+typedef unsigned int vec_size;
+
+#ifndef DIGITS_PRECISION
+    #define DIGITS_PRECISION 15
```

Comment:
Hmm, I would guess that, at this point, DIGIT_PRECISION will be defined... (Because it was defined 10 lines above)